.github/workflows/teamcity-nightly-workflow.yaml

```
@@ -14,7 +14,7 @@ on:
 
 jobs:
   rename-TC-nightly-branch:
-
+    if: github.repository == 'hashicorp/terraform-provider-google'
```

I think we need it in beta too?

```suggestion
    if: github.repository == 'hashicorp/terraform-provider-google' || github.repository == 'hashicorp/terraform-provider-google-beta'
```

---

good catch, thanks!